cumulusci.yml

```diff
@@ -1016,6 +1016,11 @@ flows:
                     field: "%%%NAMESPACE%%%Field_Mapping_Method__c"
                     value: "Data Import Field Mapping"
 
+    enable_rd2_managed:
```

Comment:
Is this supposed to be in here?

Reply:
Yes, we inadvertently broke flows in the Transport Layer and Elevate Connector that are using this (now unnecessary) flow, to avoid making changes to both of those projects I figured I'd just add this.